src/services/user.service.js

```diff
@@ -12,3 +15,33 @@ function getDetails() {
     };
     return fetch(`${config.commonConfig.$apiUrl}/users`, requestOptions).then(handleResponse);
 }
+function setting(username, profilePicture, bio, password1, password2) {
+    const registerBody = {
+        profilePicture : profilePicture,
+        username: username,
+        bio: bio,
+        password1: password1,
+        password2: password2
+    }
+    
+    const existingUsername = JSON.parse(localStorage.getItem('user')).profile.user.username;
+    const url =  stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.settings.endpoint}`, existingUsername);
+    console.log(url);
+    console.log(registerBody);
+
+
+    return axios.put(
+        url,
+        registerBody
```

Comment:
Please use authHeader as specified in that body.

Reply:
@vikas339 please do this and commit.

Reply:
done